### ISSUE TITLE
Using element.scroll_to on android causes a crash

### ISSUE THREAD

**Issue by patrik-cihal** [DioxusLabs/dioxus#3416]
**Problem**

**Steps To Reproduce**

Have an app where you want automatically scroll to an element.
Call this function to do so: element.scroll_to(ScrollBehavior::Instant).await;

**Expected behavior**

Should make the element visible by scrolling to it and continue functioning normally. But crashes instead after a short period of time.

**Environment:**
 - Dioxus version:  0.6.1
 - Rust version:   idk
 - OS info:         Arch Linux
 - App platform:    android

**Questionnaire**

**Comment by patrik-cihal** [comment]
Actually it crashes only when I unwrap it. I don't get any tracing from the device tho. I think this is an issue from my side since I'm not getting any tracing in general.

**Comment by Plebshot** [comment]
I can confirm this behavior, I can neither seem to scroll on android, nor do I get tracing output.

**Comment by DogeDark** [comment]
I don't believe we currently forward Android logs. They would be outputted in `adb logcat`
